test/system/700-play.bats

```diff
@@ -1035,3 +1035,37 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman kube restore user namespace" {
+    skip_if_remote "userns=auto is set on the server"
+
+    if is_rootless; then
+        grep -E -q "^$(id -un):" /etc/subuid || skip "no IDs allocated for current user"
+    else
+        grep -E -q "^containers:" /etc/subuid || skip "no IDs allocated for user 'containers'"
```

Comment:
All theses tests are skipped in CI btw, seems there are lof of them so we likely should fix that for our CI setup to add the containers entry (not for this PR of course).

Reply:
created https://github.com/containers/podman/issues/23383 for that